packages/core/src/internal/execution/future-processor/helpers/build-initialize-message-for.ts

```diff
@@ -40,7 +41,8 @@ export async function buildInitializeMessageFor(
         _extendBaseInitWith(
           JournalMessageType.DEPLOYMENT_EXECUTION_STATE_INITIALIZE,
           future,
-          strategy,
+          strategy.name,
+          strategy.config,
```

Comment:
When we expose the full strategy API we will have to ensure that passed arbitrary config is serializable.

Reply:
This is an addition to the journal, incase that is not immediately obvious.